src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -240,19 +240,29 @@ func (gen *PythonStartupScriptGenerator) getFlaskStartupModule() string {
 
 // Produces the gunicorn command to run the app
 func (gen *PythonStartupScriptGenerator) getCommandFromModule(module string, appDir string) string {
-	args := ""
+
+	// Default to AppService's timeout value (in seconds)
+	args := "--timeout 600"
```

Comment:
this long timeout?

Reply:
yes, that is the default that AppService currently has